internal/provisioner/cluster.go

```diff
@@ -98,13 +104,24 @@ func CreateCluster(provider, s3StateStore, size string, zones []string, logger l
 		return err
 	}
 
+	if waitForReady {
+		logger.WithField("dns", dns).Infof("waiting up to %d seconds for k8s cluster to become ready...", maxReadynessWaitSeconds)
+		err = waitForKubernetesReadyness(dns, kops, maxReadynessWaitSeconds)
+		if err != nil {
+			// Run non-silent validate one more time to log final cluster state
```

Comment:
Should we internalize this within the wait helper?

Reply:
I was thinking it made more sense to not include it in the wait helper. My thinking was that it is more flexible this way in that maybe you wanted to use the helper to wait multiple times in smaller increments with some action in between.

By keeping the option to print a final validation status outside of the helper we can keep the logs cleaner.

Thoughts?